packages/apollo-shared/src/ChangeManager/LocationEndChange.ts

```diff
@@ -42,6 +45,80 @@ export class LocationEndChange extends Change {
     }
   }
 
+  // ****** JOS KÄYTTÄÄ NESJS/COMMONia niin tulee luultavasti RUNTIME ERROR **********
+  /**
+   * Applies the required change to database
+   * @param backend - parameters from backend
+   * @returns
+   */
+  async applyToServer(backend: ServerDataStore) {
```

Comment:
The code for finding the (sub)feature by ID can be simplified, see `findById` in `features.service.ts` in the current main branch, or this commit: b54c932811096b49d4c078705cea926e7f1a9dd8

Reply:
In fact, we should probably create a `FeatureChange` class that extends `Change`, and put the code for finding a feature by ID in that class. Then `LocationEndChange` and `LocationStartChange` can both extend `FeatureChange` and share that code.

Reply:
Created `FeatureChange` and updated the `findById`.